test/integration/aws/conftest.py

```diff
@@ -5,11 +5,8 @@
 
 from importlib.metadata import version
 from test.aws.local_stack_access import AwsLocalStackAccess
-from test.aws.conftest import (
-    default_asset_id,
-    test_dummy_ami_id,
-    ec2_cloudformation_yml,
-)
+
+pytest_plugins = "test.aws.fixtures"
```

Comment:
pytest_plugins is deprecated for not-root conftest
https://docs.pytest.org/en/7.1.x/how-to/writing_plugins.html#requiring-loading-plugins-in-a-test-module-or-conftest-file